Gemfile

```diff
@@ -1,10 +1,9 @@
 source 'https://rubygems.org'
 
-
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '4.1.10'
-# Use sqlite3 as the database for Active Record
-gem 'sqlite3'
+ruby '2.2.2'
```

Comment:
just to be a stickler, latest ruby is 2.2.3 and rails is 4.2.4, might as well use the latest as a starting point if we are going to specify version.